TESTS/scripts/jinja_env/templates/declare_rom_tensor.cpp

```diff
@@ -0,0 +1 @@
+Tensor {{tensor_name}} = new RomTensor({ {%for s in shape%}{{ s }}{{"," if not loop.last}}{%endfor%} }, {{TENSOR_TYPE_MAP[tensor_type_str]}}, {{const_var_name}});
```

Comment:
should probably group these into one object, so `{{ tensor.name }}`, `{{ tensor.shape }}`, etc. It's just a little clearer 

NOTE: I am aware I didnt do it for my scripts XD

Reply:
That's ok.
I just want to keep it simple for now.